meson.build

```diff
@@ -2168,8 +2168,11 @@ userspace_c_args += ['-include', 'config.h']
 jinja2_cmdline = [meson_render_jinja2, config_h, version_h]
 
 userspace = declare_dependency(
+        sources : config_h,
         compile_args : userspace_c_args,
         link_args : userspace_c_ld_args,
+        include_directories : include_directories('.'),
+        dependencies : versiondep,
```

Comment:
`sources` and `include_directories` should be provided by versiondep, no?

Reply:
I thought specifying `sources : config_h` is safer, but that does not work, as config.h is created by meson instead of ninja. I will drop the changes.